packages/application-shell/src/components/back-to-project/back-to-project.spec.tsx

```diff
@@ -0,0 +1,29 @@
+import { mocked } from 'ts-jest/utils';
+import React from 'react';
+import { renderApp, fireEvent, wait } from '../../test-utils';
+import BackToProject from './back-to-project';
+
+beforeEach(() => {
+  window.location.replace = jest.fn();
+});
+
+describe('with `projectKey`', () => {
+  it('should redirect to the project with the key', async () => {
+    const rendered = renderApp(<BackToProject projectKey="test-project-key" />);
+    fireEvent.click(rendered.getByText('Back to project'));
+    await wait(() => {
```

Comment:
It's a bit odd that this works to me. Likely just cause it puts things on the next tick of the event queue and then the warning isn't thrown.